src/app/logic/timer.js

```diff
@@ -20,14 +20,16 @@ class GameTimer {
     let timeLeft = initialTimeInMilisec;
 
     const timer = setInterval(() => {
-      timeLeft -= timeInterval;
-      const timeLeftInSec = timeLeft / ONE_SECOND_MILLIS;
-      callbackOnInterval(timeLeftInSec, totalTimeInSec);
       if (timeLeft <= 0 || this.stopGame === true) {
         clearInterval(timer);
         timeLeft = initialTimeInMilisec;
         callbackOnEndOfTime();
```

Comment:
It is still processing the results, perhaps add a return statement or wrap the rest in else if block so it doesnt start counting again

Reply:
![1](https://user-images.githubusercontent.com/92270179/147882785-843e690c-a7e7-48e0-a9c0-01d08f9a5b7e.JPG)

Changed! if-else statement makes good work here. I tried it earlier, but I made a stupid mistake, and I didn't work then. Now it is ok!